nat-lab/tests/test_telio_version_compatibility.py

```diff
@@ -0,0 +1,102 @@
+import asyncio
+from contextlib import AsyncExitStack
+
+import pytest
+import telio
+from mesh_api import API
+from telio_features import Direct, TelioFeatures
+from utils import ConnectionTag, Ping, new_connection_with_conn_tracker, testing
+from utils.connection_tracker import (
+    ConnectionLimits,
+    generate_connection_tracker_config,
+)
+
+STUN_PROVIDER = ["stun"]
+
+UHP_conn_client_types = [
+    (
+        STUN_PROVIDER,
+        ConnectionTag.DOCKER_CONE_CLIENT_1,
+        ConnectionTag.DOCKER_CONE_CLIENT_2,
+    ),
+]
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "endpoint_providers, client1_type, client2_type",
+    UHP_conn_client_types,
+)
+async def test_connect_different_telio_version_through_relay(
+    endpoint_providers,
+    client1_type,
+    client2_type,
+) -> None:
+    async with AsyncExitStack() as exit_stack:
+        api = API()
+
+        (alpha, beta, gamma) = api.default_config_three_nodes()
```

Comment:
is there any purpose for `gamma`?

Reply:
Left for later implementation.

Reply:
adding additional node to meshmap slows workflow (I already tripped on this myself)... I'd suggest adding `TODO` or something for future implementation and only use 2 nodes for now